src/components/templates/Audience/Audience.tsx

```diff
@@ -46,6 +43,14 @@ interface ChatOutDataType {
   text: string;
 }
 
+// If you change this, make sure to also change it in Audience.scss's $seat-size
+const SEAT_SIZE = "var(--seat-size)";
+const SEAT_SIZE_MIN = "var(--seat-size-min)";
```

Comment:
[NB] As per my other comment in this review, if we think there is better wording we could use here, feel free to make it better.